src/Instances/onDestroy.lua

```diff
@@ -74,4 +74,4 @@ local function onDestroy(instanceRef: PubTypes.SemiWeakRef, callback: (...any) -
 	return disconnect
 end
 
-return onDestroy
```

Comment:
This appears on a couple of the diffs, not sure what it's meant to show (other than a lack of a newline at the end of the file) but I didn't change it? Both the addition and the removal look identical to me.